README.md

```diff
@@ -21,13 +21,6 @@ docker run \
   gcr.io/opensourcecoin/ipfs-test-network
 ```
 
-To wait until the nodes have connected to each other run
-```
-docker run ipfs-test-network /app/wait-until-ready
-```
-The command will timeout after 10 seconds and exit with status code 1.
-
-
```

Comment:
This delete seems unrelated to the change of this PR.

Reply:
Rebase issue